### PR TITLE
Airline Passenger Satisfaction

### PR DESCRIPTION
**Pull request by Soumayan-pal01**
## Airline Passenger Satisfaction

- The aim of this project is to perform analysis of the passenger satisfaction data and data preprocessing to prepare the data and predict whether the passenger is satisfied or not with the airline services.

- [x] LGM-SOC'21 Participant
- [ ] DevIncept Participant 
- [ ] Contributor

Closes: #398 

### Describe the add-ons or changes you've made

In this project we have performed a detailed analysis and visualization of the training dataset with different Exploratory Data Analysis techniques. Then a comaparative analysis of different Classifier models have been done to predict the passenegr satisfaction class. After performing the model comparative analysis we can conclude that the Random Forest Classifier and the Support Vector classifier have performed better. HYpertuning on both of these classifiers have been done to reduce overfitting and achieve an improved accuracy of 97.15% and 96.57% respectively.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by prathimacode-hub — APPROVED** [review]
Woo, this is awesome man. Proud of you. This is a killer PR. 🤩👌👌@Soumayan-pal01

**Comment by Soumayan-pal01** [comment]
> Woo, this is awesome man. Proud of you. This is a killer PR. 🤩👌👌@Soumayan-pal01

Thank you 👀@prathimacode-hub
